open-metadata-implementation/common-services/multi-tenant/src/main/java/org/odpi/openmetadata/commonservices/multitenant/OMAGServerPlatformInstanceMap.java

```diff
@@ -867,13 +867,13 @@ public  Date getServerEndTime(String  userId,
 
 
     /**
-     * Add a new service instance to the server map.
+     * Add a new service instance to the server map. It is public so it is visible for junit.
      *
      * @param serverName name of the server
      * @param serviceName name of the service running on the server
      * @param instance instance object
      */
-    void  addServiceInstanceToPlatform(String                    serverName,
+    public void  addServiceInstanceToPlatform(String             serverName,
```

Comment:
are we ok with making it exposed externally to the package?

Reply:
We talked about this - you were happy to take this as a separate issue

Reply:
I think it's just a style question to the code owner :-) I don't think we need to persue?